src/script/api/script_airport.cpp

```diff
@@ -156,9 +156,9 @@
 	return AirportGetNearestTown(as, AirportTileTableIterator(as->table[0], tile), dist)->index;
 }
 
-/* static */ uint16 ScriptAirport::GetMaintenanceCostFactor(AirportType type)
+/* static */ SQInteger ScriptAirport::GetMaintenanceCostFactor(AirportType type)
 {
-	if (!IsAirportInformationAvailable(type)) return INVALID_TOWN;
+	if (!IsAirportInformationAvailable(type)) return -1;
```

Comment:
```suggestion
	if (!IsAirportInformationAvailable(type)) return 0;
```

If `ScriptRoad::GetMaintenanceCostFactor` and `ScriptRail::GetMaintenanceCostFactor` return 0, then this one should also return 0 for consistency.